model/src/main/proto/feedback_reporting.proto

```diff
@@ -275,11 +277,12 @@ message FeedbackReportingSystemContext {
   string package_version_name = 1;
   // The package code for the specific app version on the device.
   int32 package_version_code = 2;
-  // The ISO-639 code for the country locale, as determined by the system settings. This
-  // corresponds to the country locale as set by the device settings.
+  // The ISO-639 country code for the country locale set by the user (otherwise determined by the
+  // environment JVM). EX: A locale of Canada versus French Canada suggests different contexts for
```

Comment:
Does 'different contexts' mean 'different locales' in this case? Please clarify in this comment if so.

Reply:
I meant to convey that these are two different locales and suggests two different physical contexts that a user might be in -- rephrased to be more explicit about this with a more in-depth example.